modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -387,6 +403,15 @@ private void updateLeaseBatchInternal() {
                     return;
                 }
 
+                LeaseUpdateStatisticsParameters totalLeases = leaseUpdateStatistics.onSuccessfulIteration(leasesUpdatedInCurrentIteration);
+
+                if (shouldLogLeaseStatistics) {
+                    LOG.info("Leases updated (printed once per " + LeaseUpdateStatistics.PRINT_ONCE_PER_ITERATIONS + " iterations): ["
```

Comment:
Avoid concut strings in loggers because it has an effective parametrazed method.

Reply:
fixed